### ISSUE TITLE
Haptic Feedback Automatically ON in beta-a7edc6d

### ISSUE THREAD

**Issue by Vezok74** [vcmi/vcmi#4122]
**Describe the bug**
Tested on fresh install and installation over old beta build, haptic feedback is turned on even though launcher settings say otherwise.

**To Reproduce**
1. Install beta-a7edc6d
2. Without changing any launcher settings, launch the game
3. Click from main menu to see haptic feedback

**Expected behavior**
The default setting for haptic feedback is OFF, so one shouldn't experience it on default settings

**Actual behavior**
Haptic Feedback button from launcher settings does not work until the player starts a new game or loads save game, go to settings there and click the box for haptic feedback.
 Then the button from launcher settings works as intended.

**Version**
 - OS: Android
 - Version: beta-a7edc6d

**Comment by IvanSavenko** [comment]
Haptic feedback changing to on is kind of intended
Feel free to discuss whether it should be on or off.

Will check what's wrong with launcher UI.

**Comment by Vezok74** [comment]
I wasn't really talking about preference, but the button not working correctly on first install.

Steps I took:
1. Installed VCMI Daily over earlier beta build
2. Without touching any launcher settings, launched the game
3. Seeing that there's haptic feedback when clicking from main menu, quit the game
4. Looked at the haptic feedback setting in launcher settings, which was set to OFF
5. Clicked the button to turn it ON, launched the game, clicked a few times to see haptic feedback was still on, then quit the game, only to see the launcher setting turned back to OFF
6. Only after I loaded a save game, went to settings and clicked the box for haptic feedback to off was the launcher setting button function fixed
7. Did a clean install for issue confirmation. The button did not work correctly until I changed the settings in-game.
